start-postgis.sh

```diff
@@ -2,11 +2,11 @@
 
 # This script will run as the postgres user due to the Dockerfile USER directive
 
-DATADIR="/var/lib/postgresql/10.0/main"
-CONF="/etc/postgresql/10.0/main/postgresql.conf"
-POSTGRES="/usr/lib/postgresql/10.0/bin/postgres"
-INITDB="/usr/lib/postgresql/10.0/bin/initdb"
-SQLDIR="/usr/share/postgresql/10.0/contrib/postgis-2.2/"
+DATADIR="/var/lib/postgresql/10/main"
+CONF="/etc/postgresql/10/main/postgresql.conf"
+POSTGRES="/usr/lib/postgresql/10/bin/postgres"
+INITDB="/usr/lib/postgresql/10/bin/initdb"
+SQLDIR="/usr/share/postgresql/10/contrib/postgis-2.4/"
 LOCALONLY="-c listen_addresses='127.0.0.1, ::1'"
 
 # /etc/ssl/private can't be accessed from within container for some reason
```

Comment:
@timlinux, is this workaround still necessary?  I find I don't need it in my environment.  The issue is a simple permissions issue, unrelated to the AUFS comment, and it seems to have been resolved for a while now.

I have a second branch that removes these workarounds.  Ultimately, it'd be best to one-time create a keypair+certificate (at first-start), or allow specifying a keypair+certificate somehow.

https://github.com/nmschulte/docker-postgis/tree/remove-ssl-workaround